contracts/pods/PodAdminGateway.sol

```diff
@@ -0,0 +1,179 @@
+    ///         been granted that admin priviledge on that pod
+    /// @dev Used to permission the exposure of pod admin priviledges to multiple TribeRoles
+    mapping(uint256 => mapping(AdminPriviledge => EnumerableSet.Bytes32Set))
+        private podAdminRoles;
+
+    constructor(address _core, address _memberToken) CoreRef(_core) {
+        memberToken = IMemberToken(_memberToken);
+    }
+
+    /////////////////////////     GETTERS                   ///////////////////////
+
+    /// @notice Get all TribeRoles which have a particular admin priviledge on a pod
+    function getPodAdminPriviledges(uint256 _podId, AdminPriviledge _priviledge)
+        external
+        view
+        returns (bytes32[] memory)
+    {
+        if (podAdminRoles[_podId][_priviledge].length() == 0) {
+            bytes32[] memory emptyAdmins = new bytes32[](0);
+            return emptyAdmins;
+        }
+        return podAdminRoles[_podId][_priviledge].values();
+    }
+
+    /////////////////////////     STATE CHANGING API       ////////////////////////////
+
+    ///// Grant and revoke pod admin functionality to Tribe Roles
+
+    /// @notice Grant an admin priviledge to a TribeRole
+    /// @dev Permissioned to the GOVERNOR, ROLE_ADMIN (TribalCouncil)
+    function grantPodAdminPriviledge(
+        uint256 _podId,
+        AdminPriviledge priviledge,
+        bytes32 _tribeRole
+    ) public hasAnyOfTwoRoles(TribeRoles.GOVERNOR, TribeRoles.ROLE_ADMIN) {
+        podAdminRoles[_podId][priviledge].add(_tribeRole);
+        emit GrantPodAdminPriviledge(_podId, _tribeRole);
+    }
+
+    /// @notice Revoke an admin priviledge from a TribeRole
+    /// @dev Permissioned to the GOVERNOR, ROLE_ADMIN (TribalCouncil) and GUARDIAN roles
+    function revokePodAdminPriviledge(
+        uint256 _podId,
+        AdminPriviledge priviledge,
+        bytes32 _tribeRole
+    )
+        public
+        hasAnyOfThreeRoles(
+            TribeRoles.GOVERNOR,
+            TribeRoles.ROLE_ADMIN,
+            TribeRoles.GUARDIAN
+        )
+    {
+        podAdminRoles[_podId][priviledge].remove(_tribeRole);
+        emit RevokePodAdminPriviledge(_podId, _tribeRole);
+    }
+
+    /// @notice Batch grant admin priviledges
+    function batchGrantAdminPriviledge(
+        uint256[] memory _podId,
+        AdminPriviledge[] memory priviledge,
+        bytes32[] memory _tribeRole
+    ) external hasAnyOfTwoRoles(TribeRoles.GOVERNOR, TribeRoles.ROLE_ADMIN) {
+        for (uint256 i = 0; i < _podId.length; i++) {
+            grantPodAdminPriviledge(_podId[i], priviledge[i], _tribeRole[i]);
+        }
+    }
+
+    /// @notice Batch grant admin priviledges
+    function batchRevokeAdminPriviledge(
+        uint256[] memory _podId,
+        AdminPriviledge[] memory priviledge,
+        bytes32[] memory _tribeRole
+    )
+        external
+        hasAnyOfThreeRoles(
+            TribeRoles.GOVERNOR,
+            TribeRoles.ROLE_ADMIN,
```

Comment:
I feel that we should create a new role TRIBAL_COUNCIL in TribeRoles

Reply:
And/or POD_ADMIN. ROLE_ADMIN should be purely scoped to managing roles in TribeRoles and as few other things as possible.

Reply:
Agreed with the suggestion around `POD_ADMIN` + role of `ROLE_ADMIN`. Implemented